lib/Config/Migrator.php

```diff
@@ -772,4 +774,44 @@ private function migratePurchasedProductDynamicFilters(): bool {
 
     return true;
   }
+
+  private function migratePurchasedInCategoryDynamicFilters(): bool {
+    global $wpdb;
+    // skip the migration if the DB version is higher than 3.74.3 or is not set (a new install)
+    if (version_compare($this->settings->get('db_version', '3.74.3'), '3.74.2', '>')) {
```

Comment:
This PR won't make it to 3.74.3 so we will need to increase the versions in this check to 3.74.4 respectively 3.74.3.
So that we compare against the previous released version 3.74.3 and we use 3.74.4 as a default value in case of a new install.

Reply:
Noted

Thank you @costasovo 🙌🏿